swift-sdk/ITBConsts.swift

```diff
@@ -34,6 +34,7 @@ public extension String {
 
 // Keys
 public extension AnyHashable {
+    static let ITBL_HEADER_API_KEY = "Api-Key"
```

Comment:
What's the effect of this change? What behavior do old sdks experience when using the old header?

Reply:
It's for RFC compliance, so there's no technical difference between the before and after of this change (since the backend has supported both for a while).